### ISSUE TITLE
File.Create throws exception for relative file paths

### ISSUE THREAD

**Issue by Gav-Brown** [System-IO-Abstractions/System.IO.Abstractions#401]
#### Package Version

2.1.0.256

#### Repro steps

1. Create `MockFileSystem` 
2. Create working directory
3. Set working directory
4. Call
 
`using (var file = fileSystem.File.Create("File.txt"))
{
}`

#### Expected behavior

File is created in current working directory (which does exist).

#### Actual behavior

Exception is thrown.
Could not find a part of the path 'File.txt'.

This worked in version 2.0.0.144. I suspect it is due to change for #278 an #286
which assumes an absolute file path rather than a relative one.

.NET Framework version handles both cases, as does 

`fileSystem.File.WriteAllText("File.txt", null);`

**Comment by fgreinacher** [comment]
Thanks for reporting this! Could you test if the problem also occurs in [the latest prerelease version](https://www.nuget.org/packages/System.IO.Abstractions/2.2.8-beta)? #372 might already have fixed it.

**Comment by Gav-Brown** [comment]
Thanks. Alas it still throws in latest beta.

```
            string workingDirectory = @"C:\folder";
            string relativeFile = "file.txt";
            var fileSystem = new MockFileSystem();

            fileSystem.Directory.CreateDirectory(workingDirectory);
            fileSystem.Directory.SetCurrentDirectory(workingDirectory);

            // Throws.
            using(fileSystem.File.Create(relativeFile))
            {
            }

            // Works.
            fileSystem.File.WriteAllText(relativeFile, null);
```

**Comment by ThomasBarnekow** [comment]
This seems to be related to my issue #404. 

The funny thing is that issue 404 is related to something that is not found ;-)

**Comment by fgreinacher** [comment]
@Gav-Brown The problem is fixed with ec1b673. Could you verify your use case with the prerelease version https://www.nuget.org/packages/System.IO.Abstractions/2.2.10-beta?

**Comment by Gav-Brown** [comment]
@fgreinacher Thanks. I've tested and can confirm this is now resolved.
